install/resources/mybb_theme.xml

```diff
@@ -11354,6 +11356,8 @@ if(use_xmlhttprequest == "1")
 	var quickdelete_confirm = "{$lang->quickdelete_confirm}";
 	var quickrestore_confirm = "{$lang->quickrestore_confirm}";
 	var allowEditReason = "{$mybb->settings['alloweditreason']}";
+	var thread_deleted = "{$thread_deleted}";
+	var visible_replies = "{$thread['replies']}";
```

Comment:
It seems this ignores unapproved and soft deleted posts. Is this how it should be?

Reply:
Yes, as you guided earlier... Moderators can operate threads with those states...

Reply:
Oh, right, but not all moderators are able to see all posts:
https://github.com/mybb/mybb/blob/feature/showthread.php#L118-L143

As tools building doesn't take into account these conditions to build tools:
https://github.com/mybb/mybb/blob/feature/showthread.php#L1319

Reply:
I guess I got what you said. I will review this further ...

Reply:
I think this is resolved.